forge-gui/res/cardsfolder/upcoming/topiary_stomper.txt

```diff
@@ -0,0 +1,11 @@
+Name:Topiary Stomper
+ManaCost:1 G G
+Types:Creature Plant Dinosaur
+PT:4/4
+K:Vigilance
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSearch | TriggerDescription$ When CARDNAME enters the battlefield, search your library for a basic land card, put it onto the battlefield tapped, then shuffle.
+SVar:TrigSearch:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Basic | ChangeNum$ 1 | Tapped$ True
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't attack or block. | CheckSVar$ X | SVarCompare$ LT7 | Description$ CARDNAME can't attack or block unless you control seven or more lands.
+SVar:X:Count$Valid Land.YouCtrl

```

Comment:
line 8 use IsPresent and PresentCompare (see Crash of Rhino Beetles)
then you can eliminate line 9